azurelinuxagent/agent.py

```diff
@@ -151,6 +151,23 @@ def show_configuration(self):
         for k in sorted(configuration.keys()):
             print("{0} = {1}".format(k, configuration[k]))
 
+    def collect_logs(self, log_collector_mode):
+        from azurelinuxagent.common.logcollector import LogCollector, OUTPUT_RESULTS_FILE_PATH
+        if log_collector_mode == "full":
+            print("Running log collector mode full")
+        else:
+            print("Running log collector mode normal")
+
+        try:
+            lc = LogCollector(log_collector_mode == "full")
```

Comment:
In the vein of the above comment, is it worth passing a constant instead of a bool here?